com.hperrin.StreamOverlay.metainfo.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.hperrin.StreamOverlay</id>
+  <launchable type="desktop-id">com.hperrin.StreamOverlay.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+
+  <name>Stream Overlay</name>
+
+  <categories>
+    <category>Utility</category>
+  </categories>
+
+  <summary><![CDATA[Show transparent, click-through browser window overlays on screen. Like your Twitch chat and alerts on top of your game.]]></summary>
```

Comment:
this summary is invalid.

Reply:
What do you mean?

Reply:
`<![CDATA[` etc.

Reply:
I did that so it wouldn't be autoformatted and have new lines inserted. Isn't that valid for any XML document?